src/CritCore/Translations/Translations.js

```diff
@@ -244,7 +244,17 @@ let translations = {
       alterEgoTooltip: 'Nombre visible / Alter ego',
       icon: 'Elija un avatar',
       submit: 'Crear',
-      selectPlaceholder: 'Elija un ícono'
+      selectPlaceholder: 'Elija un ícono',
+      errors: {
+        email: 'Email inválido',
+        emailMinMaxLength: 'Caracteres: Min 6 | Max 60',
```

Comment:
Set this numbers from the config, that way the translations will adjust dynamically. The same for username lenght and their respective english equivalents.

Reply:
Done.